pkgs/development/python-modules/aiohomekit/default.nix

```diff
@@ -0,0 +1,56 @@
+{ lib
+, buildPythonPackage
+, cryptography
+, fetchFromGitHub
+, poetry
+, pytest-aiohttp
+, pytestCheckHook
+, pythonAtLeast
+, zeroconf
+}:
+
+buildPythonPackage rec {
+  pname = "aiohomekit";
+  version = "0.2.60";
+  format = "pyproject";
+  disabled = pythonAtLeast "3.9";
+
+  src = fetchFromGitHub {
+    owner = "Jc2k";
+    repo = pname;
+    rev = version;
+    sha256 = "03llk5i22hq163x568kz0qar5h0sda8f8cxbmgya6z2dcxv0a83p";
+  };
+
+  nativeBuildInputs = [ poetry ];
```

Comment:
Another case for `poetry-core` dance, you know how it goes.

https://github.com/NixOS/nixpkgs/issues/103325